store/src/java/com/zimbra/cs/service/account/GetInfo.java

```diff
@@ -176,6 +177,8 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
             Server server = prov.getLocalServer();
             if (server != null) {
                 response.addAttribute(AccountConstants.A_DOCUMENT_SIZE_LIMIT, server.getFileUploadMaxSize());
+                //For ZBUG-1280: checks if "spell" service is available from the list of enabled services on the server.
```

Comment:
Fix a comment. "spell" service is no longer checked.

Reply:
updated.